content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
```

Comment:
take out Cloud SIEM to align with the other guides - just "Risk Based Entity Insights"